### DISCUSSION TITLE
Globalization

### DISCUSSION THREAD

**Discussion by shane101978** [Royal-Modding/FS19_AdvancedStats#4] [Ideas]
Is it possible to change this mod to display the units in an American format? (Liters to gallons)

**Comment by TyKonKet** [comment]
Yes it is, do you need some other unit conversions?
Do you think that would be fine if I 'chain' that conversions to the in game kmh/mph setting so that if you use mph you will also have gallons and so on?

**Reply by shane101978** [reply]
That would be great, Thanks
